examples/lfric/eg11/Makefile

```diff
@@ -31,16 +31,60 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # ------------------------------------------------------------------------------
-# Author: A. R. Porter, STFC Daresbury Laboratory
-# Modified J. Henrichs, Bureau of Meteorology
+# Authors: A. R. Porter, STFC Daresbury Lab
+#          J. Henrichs, Bureau of Meteorology
+# Modifications: R. W. Ford, STFC Daresbury Lab
+
+# The compiler to use may be specified via the F90 environment
+#
+# export F90=gfortran
+# export F90FLAGS="-g -O0"
 
 include ../../common.mk
 
-transform:
-	${PSYCLONE} -s ./async_script.py helmholtz_solver_alg_mod.x90
+GENERATED_FILES = helmholtz_solver_alg_psy.f90 helmholtz_solver_alg_mod.f90 *.o *.mod $(EXEC)
+
+F90 ?= gfortran
+F90FLAGS ?= -g -O0
+
+OBJ = helmholtz_solver_alg_psy.o helmholtz_solver_alg_mod.o scaled_matrix_vector_kernel_mod.o
 
-compile: transform
-	@echo "No compilation supported for lfric/eg11"
+EXEC = compiles
+LFRIC_PATH = ../../../src/psyclone/tests/test_files/dynamo0p3/infrastructure
+LFRIC_NAME=lfric
+LFRIC_LIB=$(LFRIC_PATH)/lib$(LFRIC_NAME).a
+
+F90FLAGS += -I$(LFRIC_PATH)
 
 run: compile
```

Comment:
Please add a comment explaining why it won't run.

Reply:
Done.